todo_app/templates/layout.html

```diff
@@ -20,5 +21,7 @@
     <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+    <script src={{ url_for('static', filename="js/bootstrap-datepicker.js")}}></script>
+    <script src={{ url_for('static', filename="js/bootstrap-datepicker.min.js")}}></script>
```

Comment:
No reason to load both - the .min file is a "minified" version of the other one. No functional difference, just a smaller file by removing whitespace/renaming variables.

And I think you can remove the other js files